build.gradle

```diff
@@ -20,12 +22,16 @@ plugins {
     id "com.github.kt3k.coveralls" version "2.8.2"
     id "com.github.spotbugs" version "1.6.6"
     id 'io.franzbecker.gradle-lombok' version '2.0'
+    id 'net.ltgt.apt' version '0.21'
```

Comment:
Do designer need `net.ltgt.apt' version '0.21'`

Reply:
agreed, removed that one